### ISSUE TITLE
CWD for unit test execution should match VS.NET or be configurable

### ISSUE THREAD

**Issue by gerrard00** [OmniSharp/omnisharp-vscode#607]
## Environment data

`dotnet --info` output:

```
.NET Command Line Tools (1.0.0-preview2-003121)

Product Information:
 Version:            1.0.0-preview2-003121
 Commit SHA-1 hash:  1e9d529bc5

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.11
 OS Platform: Darwin
 RID:         osx.10.11-x64
```

VS Code version: 1.3.1
C# Extension version: 1.2.2
## Steps to reproduce
1. Open a folder that contains a solution in VS Code
2. Locate an xunit test function that uses a relative path
3. Click "run test" or "debug test"
## Expected  behavior

The current working directory for the execution should be the test project's root folder, which matches the behavior of Visual Studio .NET. That would allow relative paths to work correctly. If not automatically set, this should be configurable.
## Actual behavior

The current working directory is the `workspaceRoot` as set in [this line of the dotnetTest.ts file](https://github.com/OmniSharp/omnisharp-vscode/blob/39d4ceb79be350274faf9c89285270f979d331d3/src/features/dotnetTest.ts#L66).

This makes it hard to run tests which were written by devs working in Visual Studio .NET which use relative paths. I have a workaround now, but it requires modifying every test to use my helper function.

**Comment by gregg-miskelly** [comment]
cc @troydai

**Comment by eamodio** [comment]
It seems that for the `run test` (via CodeLens) the path is correctly set to the path of the test project root, but for `debug test` (again via CodeLens) the path seems to be set to the workspace root.

**Comment by DustinCampbell** [comment]
Yes, this is a bug. 😞

**Comment by DustinCampbell** [comment]
This was fixed in 1.9: https://github.com/OmniSharp/omnisharp-vscode/blob/master/src/features/dotnetTest.ts#L73-L86.
